tools/notebook/postprocess.py

```diff
@@ -90,7 +90,15 @@ def postprocess_notebooks(input_dir, output_base_dir):
     import os
     import glob
     from nbformat import read, write, NO_CONVERT
-    notebooks = [(os.path.split(nbfile)[-1], read(nbfile, NO_CONVERT))
+
+    def _read(nbfile, to_conv):
+        try:
+            return read(nbfile, to_conv)
+        except ValueError as e:
```

Comment:
Might as well catch ``Exception as e`` since you might run into errors besides just ``ValueError`` :-)

Reply:
Makes sense. The added information would be relevant regardless.